kratos/solving_strategies/schemes/residual_based_bossak_displacement_scheme.hpp

```diff
@@ -194,41 +198,42 @@ class ResidualBasedBossakDisplacementScheme
      * @details Incremental update within newton iteration. It updates the state variables at the end of the time step u_{n+1}^{k+1}= u_{n+1}^{k}+ \Delta u
```

Comment:
at the end of the time step or at the end of the non-linear iteration step ?

Reply:
Update->End iteration
Predict->Before step